lib/awful/popup.lua

```diff
@@ -453,8 +465,8 @@ end
 -- @tparam string|table args.preferred_positions
 -- @tparam string|table args.preferred_anchors
 -- @tparam table|number args.offset The X and Y offset compared to the parent object
--- @tparam boolean args.hide_on_right_click Whether or not to hide the popup on
---  right clicks.
+-- @tparam boolean args.hide_on_click Whether or not to hide the popup on
```

Comment:
what about turning it from boolean into int, so user could choose the mouse button?